test/benchmark/README.md

```diff
@@ -242,7 +242,8 @@ The `bench_result.json` file configures how benchmark results are processed and
     // Settings for extracting benchmark results from raw outputs.
     "result_extraction": {
         "search_pattern": "sender", // Regex or string to locate results.
-        "result_index": 7 // Match index to use (e.g., 7th occurrence).
+        "nth_occurrence": 1, // Which matched occurrence to use (default to 1).
+        "result_index": 7 // Match index to use.
```

Comment:
I think a more logical ordering is to put `"search_pattern"` and `"result_index"` together and before `"nth_occurrence"`.

```json
    "result_extraction": {
        "search_pattern": "sender", // Regex or string to locate results.
        "result_index": 7 // Match index to use (e.g., 7th occurrence).
        "nth_occurrence": 1, // Which matched occurrence to use (default to 1).
    }
```